topics/admin/tutorials/reference-genomes/tutorial.md

```diff
@@ -0,0 +1,667 @@
+> > >                 </column>
+> > >             </output>
+> > >         </data_table>
+> > >     </data_manager>
+> > > ```
+> > {: .code-out.code-max-300}
+> >
+> > > <code-in-title>Bash</code-in-title>
+> > > Let's also investigate the tool data table config file.
+> > > ```bash
+> > > cat /srv/galaxy/var/config/shed_tool_data_table_conf.xml
+> > > ```
+> > {: .code-in}
+> >
+> > > <code-out-title>Bash</code-out-title>
+> > > ```xml
+> > > <?xml version="1.0" ?>
+> > > <tables>
+> > >     <table name="all_fasta" comment_char="#">
+> > >         <columns>value, dbkey, name, path</columns>
+> > >         <file path="/srv/galaxy/var/tool-data/toolshed.g2.bx.psu.edu/repos/devteam/data_manager_fetch_genome_dbkeys_all_fasta/4d3eff1bc421/all_fasta.loc"/>
+> > >         <tool_shed_repository>
+> > >             <tool_shed>toolshed.g2.bx.psu.edu</tool_shed>
+> > >             <repository_name>data_manager_fetch_genome_dbkeys_all_fasta</repository_name>
+> > >             <repository_owner>devteam</repository_owner>
+> > >             <installed_changeset_revision>4d3eff1bc421</installed_changeset_revision>
+> > >         </tool_shed_repository>
+> > >     </table>
+> > >     <table name="__dbkeys__" comment_char="#">
+> > >         <columns>value, name, len_path</columns>
+> > >         <file path="/srv/galaxy/var/tool-data/toolshed.g2.bx.psu.edu/repos/devteam/data_manager_fetch_genome_dbkeys_all_fasta/4d3eff1bc421/dbkeys.loc"/>
+> > >         <tool_shed_repository>
+> > >             <tool_shed>toolshed.g2.bx.psu.edu</tool_shed>
+> > >             <repository_name>data_manager_fetch_genome_dbkeys_all_fasta</repository_name>
+> > >             <repository_owner>devteam</repository_owner>
+> > >             <installed_changeset_revision>4d3eff1bc421</installed_changeset_revision>
+> > >         </tool_shed_repository>
+> > >     </table>
+> > > </tables>
+> > > ```
+> > {: .code-out.code-max-300}
+> >
+> {: .solution }
+>
+{: .question}
+
+## Download and install a reference genome sequence
+
+Next, we will install some reference data. Specifically, we will grab sacCer2 (version 2 of the Saccharomyces cerevisiae genome).
+
+> <hands-on-title>Download and install sacCer2</hands-on-title>
+>
+> 1. Access the Admin menu from the top bar
+> 2. Click **Local Data**, which can be found on the left, under **Server**
+>
+>    You should see something like this:
+>
+>    ![nearly empty data manager tool list](../../images/dm-admin-local-data.png)
+>
+>    If you instead see the message "You do not currently have any Data Managers installed," you may need to restart your Galaxy server.
+>
+> 3. Click **all_fasta** under **View Tool Data Table Entries**
+>
+>    You should see the current contents of `tool-data/all_fasta.loc`, which will be empty.
+>
+> 4. Click **Local Data** to return to the previous page
+>
+> 5. Click {% tool [Create DBKey and Reference Genome - fetching]({{data_manager_fetch_genome_all_fasta_dbkey}}) %} under **Installed Data Managers**.
```

Comment:
```suggestion
> 5. Click {% tool [Create DBKey and Reference Genome - fetching](data_manager_fetch_genome_all_fasta_dbkey) %} under **Installed Data Managers**.
```

as far as I can tell this isn't a variable? If it is a variable there's a "NOQA" type statement we can add to this tutorial.

Reply:
Definitely not a variable, this was me probably miscopying the syntax for tool links from another tutorial.